Publish all libraries as pip packages

JonWBedard opened the pull request:
#### fa975f227436bcd23e7468cb2060c5bdab6e65ea
<pre>
Publish all libraries as pip packages
<a href="https://bugs.webkit.org/show_bug.cgi?id=281766">https://bugs.webkit.org/show_bug.cgi?id=281766</a>
<a href="https://rdar.apple.com/138198594">rdar://138198594</a>

Reviewed by Sam Sneddon.

Add LICENSE files to and correct description formats to every one of our Python libraries.

* Tools/Scripts/libraries/reporelaypy/LICENSE: Added.
* Tools/Scripts/libraries/reporelaypy/README.md: Added.
* Tools/Scripts/libraries/reporelaypy/setup.py:
* Tools/Scripts/libraries/resultsdbpy/LICENSE: Added.
* Tools/Scripts/libraries/resultsdbpy/setup.py:
* Tools/Scripts/libraries/webkitbugspy/LICENSE: Added.
* Tools/Scripts/libraries/webkitbugspy/setup.py:
* Tools/Scripts/libraries/webkitbugspy/webkitbugspy/__init__.py:
* Tools/Scripts/libraries/webkitcorepy/LICENSE: Added.
* Tools/Scripts/libraries/webkitcorepy/setup.py:
* Tools/Scripts/libraries/webkitcorepy/webkitcorepy/__init__.py:
* Tools/Scripts/libraries/webkitflaskpy/LICENSE: Added.
* Tools/Scripts/libraries/webkitflaskpy/setup.py:
* Tools/Scripts/libraries/webkitflaskpy/webkitflaskpy/__init__.py:
* Tools/Scripts/libraries/webkitscmpy/LICENSE: Added.
* Tools/Scripts/libraries/webkitscmpy/setup.py:

Canonical link: <a href="https://commits.webkit.org/285459@main">https://commits.webkit.org/285459@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/47c8b5d8540a543921dd52b7fc5026b8f6ad5974

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/72607 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/52032 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/25405 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/76795 "Built successfully") | [  ~~🛠 win~~](https://ews-build.webkit.org/#/builders/59/builds/23878 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/74722 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/59837 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/23641 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/57139 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/59/builds/23878 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/75674 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/47083 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/62515 "Passed tests") | [❌ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/37566 "Found 1 new API test failure: /WPE/TestAuthentication:/webkit/Authentication/authentication-empty-realm (failure)") | 
| [✅ 🧪 webkitpy](https://ews-build.webkit.org/#/builders/6/builds/72114 "Passed tests") | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/43733 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/19982 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/22170 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/65594 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/20341 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/78466 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/16853 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/62/builds/19479 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/65586 "Passed tests") | 
| [✅ 🧪 services](https://ews-build.webkit.org/#/builders/28/builds/72282 "Passed tests") | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/16901 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/62524 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/64855 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/13149 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/6796 "Passed tests") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/11170 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/47830 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/48897 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/50192 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/48642 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

gsnedders reviewed (approved):
This is largely the same as #17306, albeit without supporting depending on installing them locally from adjacent directories.

The only notable difference is https://github.com/WebKit/WebKit/pull/17306/files#diff-5a1af6bb13792d551726bc2c281c9bb684cce5508814bc6d6264d24f139d5f2c, but that appears to have been done previously in https://github.com/WebKit/WebKit/commit/db7018ee6d1da6f4380a1f0de64a4bac81f2b8cc.

JonWBedard committed:
Publish all libraries as pip packages
https://bugs.webkit.org/show_bug.cgi?id=281766
rdar://138198594

Reviewed by Sam Sneddon.

Add LICENSE files to and correct description formats to every one of our Python libraries.

* Tools/Scripts/libraries/reporelaypy/LICENSE: Added.
* Tools/Scripts/libraries/reporelaypy/README.md: Added.
* Tools/Scripts/libraries/reporelaypy/setup.py:
* Tools/Scripts/libraries/resultsdbpy/LICENSE: Added.
* Tools/Scripts/libraries/resultsdbpy/setup.py:
* Tools/Scripts/libraries/webkitbugspy/LICENSE: Added.
* Tools/Scripts/libraries/webkitbugspy/setup.py:
* Tools/Scripts/libraries/webkitbugspy/webkitbugspy/__init__.py:
* Tools/Scripts/libraries/webkitcorepy/LICENSE: Added.
* Tools/Scripts/libraries/webkitcorepy/setup.py:
* Tools/Scripts/libraries/webkitcorepy/webkitcorepy/__init__.py:
* Tools/Scripts/libraries/webkitflaskpy/LICENSE: Added.
* Tools/Scripts/libraries/webkitflaskpy/setup.py:
* Tools/Scripts/libraries/webkitflaskpy/webkitflaskpy/__init__.py:
* Tools/Scripts/libraries/webkitscmpy/LICENSE: Added.
* Tools/Scripts/libraries/webkitscmpy/setup.py:

Canonical link: https://commits.webkit.org/285459@main
